src/main/scala/util/DescribedSRAM.scala

```diff
@@ -26,8 +28,9 @@ object DescribedSRAM {
     name: String,
     desc: String,
     size: BigInt, // depth
-    data: T
-  ): SyncReadMem[T] = {
+    data: T,
+    uid: Int = -1
```

Comment:
Why are you allowing it to be passed in here when you ignore it below?

Reply:
fixed